Magento2/Sniffs/Namespaces/UseDeclarationSniff.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
I'd like to have `UseDeclarationSniff.md` file that will describe the issue behind